src/SFA.DAS.ProviderRequestApprenticeTraining.Web/Controllers/HomeController.cs

```diff
@@ -32,10 +31,10 @@ public IActionResult Index()
             return Redirect(_configuration.DashboardUrl);
         }
 
-        [HttpGet("{ukprn}/start", Name = StartRouteGet)]
+        [HttpGet("start", Name = StartRouteGet)]
         public IActionResult Start()
         {
-            return View();
+            return RedirectToRoute(EmployerRequestController.ActiveRouteGet);
```

Comment:
And here including the ukprn which is in the claim in this method which is just a debug only method

Reply:
Done